modules/openy_gc_auth/modules/openy_gc_auth_daxko_sso/src/Plugin/GCIdentityProvider/DaxkoSSO.php

```diff
@@ -120,6 +120,14 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
       '#required' => TRUE,
     ];
 
+    $form['error_accompanying_message'] = [
+      '#title' => $this->t('Authentication error message'),
+      '#description' => $this->t('Message displayed to user when he failed to log in using personify plugin.'),
```

Comment:
Description still references Personify, should be Daxko

Reply:
done